tests/functional/spec/features/calculated_summary/new_calculated_summary_repeating_and_static_answers.spec.js

```diff
@@ -173,8 +173,28 @@ describe("Calculated summary with repeating answers", () => {
     await assertSummaryValues(["3", "7"]);
     await $(CalculatedSummaryVisitsPage.submit()).click();
 
-    await expect(await $(Section1SummaryPage.supermarketsListLabel(1)).getText()).to.equal("Lidl");
-    await expect(await $(Section1SummaryPage.supermarketsListLabel(2)).getText()).to.equal("Sainsburys");
-    await expect(await $(Section1SummaryPage.supermarketsListLabel(3)).isExisting()).to.be.false;
+    await expect(await $(SummaryPage.supermarketsListLabel(1)).getText()).to.equal("Lidl");
+    await expect(await $(SummaryPage.supermarketsListLabel(2)).getText()).to.equal("Sainsburys");
+    await expect(await $(SummaryPage.supermarketsListLabel(3)).isExisting()).to.be.false;
+  });
+
+  it("Given I proceed to the second section, the calculated summary of days I visit the supermarket is used as a maximum to the first question", async () => {
```

Comment:
Description would probably need to change to something like "...Then the correct error message is displayed" or we need to change the assertions? Check `guidancePanel(guidanceIndex)` or whatever it is that shows the right text?

Reply:
Changed 👍